### PR TITLE
Added: alt attribute text to few images

### PR DESCRIPTION
**Pull request by chaitanyaj2222**
#### Summary

* I have added alt attributes to a handful of items in the content
#### Motivation

* Adding the alt attribute to the image displays the value of `alt` attribute when the respective `image fails to load` rather than leaving the space blank. It is also read aloud by the screen readers when there is an image to describe the user about the image.
#### Supporting details

@estelle  has collected the list of images from the documentation that do not have the alt attribute. [link to the spreadsheet](https://docs.google.com/spreadsheets/d/1cDHee6mwDMlY6zANEmg8MjR8tL-wrCS86uawpWZ4XbA/edit#gid=0)
#### Metadata

- [ ] Adds a new document
- [X] Rewrites (or significantly expands) a document
- [ ] Fixes a typo, bug, or other error

### CODE REVIEW

**Comment by Chaitanya7666** [comment]
#19334

File: files/en-us/web/api/canvas_api/tutorial/compositing/index.md [hunk 1]
```diff
@@ -29,7 +29,7 @@ See [compositing examples](/en-US/docs/Web/API/Canvas_API/Tutorial/Compositing/E
 
 A clipping path is like a normal canvas shape but it acts as a mask to hide unwanted parts of shapes. This is visualized in the image below. The red star shape is our clipping path. Everything that falls outside of this path won't get drawn on the canvas.
 
-![](canvas_clipping_path.png)
+![A canvas co-ordinate with a star outlined in red color. The grid squares lying inside the star are clearly visible whereas the grid squares lying outside the star are blurred. The center of the canvas is at the left-hand side top corner. X-axis is the horizontal line at the top. Y-axis is the horizontal line at the left.](canvas_clipping_path.png)
```
Comment (estelle) [inline files/en-us/web/api/canvas_api/tutorial/compositing/index.md:32]:
```suggestion
![A canvas with a star outlined in red color. The inside of the star is transparent, as portrayed by the grid squares inside the star being clearly visible whereas the grid squares lying outside the star are blurred. ](canvas_clipping_path.png)
```

**Review by estelle — APPROVED** [review]
Thank you! This is so helpful.

File: files/en-us/learn/server-side/express_nodejs/forms/create_bookinstance_form/index.md [hunk 2]
```diff
@@ -101,7 +101,7 @@ exports.bookinstance_create_post = [
       if (err) {
         return next(err);
       }
-      
+
```
Comment (estelle) [inline files/en-us/learn/server-side/express_nodejs/forms/create_bookinstance_form/index.md:104]:
```suggestion
```

**Comment by Chaitanya7666** [comment]
Thank you!

**Pull request merged**